index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href = "style.css">
+        <title>Tamashevich CV</title>
+        </title>
+    </head>
+    <body>
+        <div>
+            <h1>Sviatlana Tamashevich</h1>
+        </div>
+        <div>
+            <h2>Junior software developer</h2>
+        </div>
+        <img src = "photo.jpg" alt = "photo" />
```

Comment:
Let's don't use spaces nearby `=` in element attributes, like this `<img src="photo.jpg" alt="photo" />`